armi/reactor/tests/test_reactors.py

```diff
@@ -743,7 +744,10 @@ def test_createAssemblyOfType(self):
         # creation with modified enrichment on an expanded BOL assem.
         fuelComp = fuelBlock.getComponent(Flags.FUEL)
         bol = self.r.blueprints.assemblies[aOld.getType()]
-        bol.axiallyExpand(0.05, fuelComp.getNuclides())
+        obj = AxialExpansionChanger(converterSettings={})
```

Comment:
Typically it's good to call this a more explicit name like `changer =` for clarity

Reply:
Addressed.